File: source/kCura.Relativity.Export.ConvertedToC#/Types/LoadFileType.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Types
{
	public class LoadFileType
	{
		public enum FileFormat
		{
			Opticon = 0,
			IPRO = 1,
			IPRO_FullText = 2
		}

		public static System.Data.DataTable GetLoadFileTypes()
		{
			System.Data.DataTable dt = new System.Data.DataTable();
			dt.Columns.Add("DisplayName");
			dt.Columns.Add("Value", typeof(Int32));
			dt.Rows.Add(new object[] {
				"Select...",
				-1
			});
			dt.Rows.Add(new object[] {
				"Opticon",
				0
			});
			dt.Rows.Add(new object[] {
				"IPRO",
				1
			});
			dt.Rows.Add(new object[] {
				"IPRO (FullText)",
				2
			});
			return dt;
		}
	}
}

File: source/kCura.Relativity.Export/Types/LoadFileType.vb
Namespace kCura.Relativity.Export.Types
	Public Class LoadFileType
		Public Enum FileFormat
			Opticon = 0
			IPRO = 1
			IPRO_FullText = 2
		End Enum

		Public Shared Function GetLoadFileTypes() As System.Data.DataTable
			Dim dt As New System.Data.DataTable
			dt.Columns.Add("DisplayName")
			dt.Columns.Add("Value", GetType(Int32))
			dt.Rows.Add(New Object() {"Select...", -1})
			dt.Rows.Add(New Object() {"Opticon", 0})
			dt.Rows.Add(New Object() {"IPRO", 1})
			dt.Rows.Add(New Object() {"IPRO (FullText)", 2})
			Return dt
		End Function
	End Class
End Namespace